sections/community_brokers.tex

```diff
@@ -27,19 +27,21 @@ \subsection{Requirements for Community Brokers}
 
 Because of the large data volume of the alert stream (several TB per night), finite bandwidth from the LSST datacenter prevents sending a full copy of the alert stream to all interested parties.
 Additionally, we anticipate that community brokers will require some level of support from the LSST operations team.
-Accordingly, LSST will conduct an open proposal process in order to select brokers with sufficient technical capability to enable valuable and reliable scientific returns.
+Accordingly, LSST \oldtext{will}\newtext{is} conduct\newtext{ing} an open proposal process in order to select brokers with sufficient technical capability to enable valuable and reliable scientific returns.
 
 We believe the scientific community will be best served by a rich broker ecosystem offering diverse capabilities.
-Any institution worldwide may submit a Letter of Intent (\S \ref{sec:LOIs}), and if invited, a full proposal.
-Demonstration of appropriate technical and personnel resources to support the proposed goals will be the minimum requirement to be considered.
+Any institution worldwide \oldtext{may}\newtext{was allowed to} submit a Letter of Intent (\S \ref{sec:LOIs}), and if invited, a full proposal.
```

Comment:
Any institution worldwide \oldtext{may}\newtext{was allowed to} submit a Letter of Intent (\S \ref{sec:LOIs}). The LSST SAC decided to invite all institutions who submitted a LOI to submit a full proposal.